src/scripts/KeyListeners.ts

```diff
@@ -1,7 +1,7 @@
 $(document).on("keydown", function (e) {
     let keyCode = e.keyCode;
 
-    if (Game.gameState() == GameConstants.GameState.dungeon) {
+    if (App.game.gameState == GameConstants.GameState.dungeon) {
```

Comment:
Oh, there is a KeyListeners file. What is the difference between this and KeyBinds? (Other than doing the same thing for a different feature)
Maybe something to deal with Another Time.

Reply:
No idea, seems both perfect to go into the GameController class